core-web/libs/block-editor/src/lib/extensions/action-button/actions-menu.extension.ts

```diff
@@ -209,16 +243,18 @@ export const ActionsMenu = (viewContainerRef: ViewContainerRef) => {
 
     function setUpSuggestionComponent(editor: Editor, range: Range) {
         const { allowedBlocks, allowedContentTypes, lang } = editor.storage.dotConfig;
+        const editorAllowedBlocks = allowedBlocks.length > 1 ? allowedBlocks : [];
+        const items = getItems({ allowedBlocks: editorAllowedBlocks, editor, range });
+
         suggestionsComponent = viewContainerRef.createComponent(SuggestionsComponent);
 
         // Setting Inputs
+        suggestionsComponent.instance.items = [...items];
```

Comment:
If I am not mistaken, you are creating a new array but the internal objects will have the same ref.